Index: dwm-patches/dwm/config.def.h
===================================================================
--- dwm-patches.orig/dwm/config.def.h
+++ dwm-patches/dwm/config.def.h
@@ -71,6 +71,8 @@ static const char *volumedown[] = { "ami
 static const char *volumeup[]   = { "amixer", "set", "Master", "5%+", NULL };
 static const char *volumemute[] = { "amixer", "set", "Master", "toggle", NULL };
 
+#include "shifttags.c"
+
 static Key keys[] = {
 	/* modifier                     key        function        argument */
 	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
@@ -115,6 +117,8 @@ static Key keys[] = {
 	{ 0,                       0x1008ff13,     spawn,          {.v = volumeup } },
 	{ 0,                       0x1008ff11,     spawn,          {.v = volumedown } },
 	{ 0,                       0x1008ff12,     spawn,          {.v = volumemute } },
+	{ MODKEY|ShiftMask,             XK_j,      shifttags,      {.i = -1} },
+	{ MODKEY|ShiftMask,             XK_k,      shifttags,      {.i = +1} },
 };
 
 /* button definitions */
