Index: dwm-patches/dwm/dwm.c
===================================================================
--- dwm-patches.orig/dwm/dwm.c
+++ dwm-patches/dwm/dwm.c
@@ -249,7 +249,9 @@ static void (*handler[LASTEvent]) (XEven
 	[ConfigureRequest] = configurerequest,
 	[ConfigureNotify] = configurenotify,
 	[DestroyNotify] = destroynotify,
-	[EnterNotify] = enternotify,
+	#if 0 /* disable focus follows mouse behaviour */
+		[EnterNotify] = enternotify,
+	#endif
 	[Expose] = expose,
 	[FocusIn] = focusin,
 	[KeyPress] = keypress,
